capstone-backend/src/main/java/com/google/sps/data/BookList.java

```diff
@@ -0,0 +1,73 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+/**
+ * The Book List object stores userID and bookshelfID.
+ * 
+ * These IDs variables are meant to be fetch from the Frontend, to extract a
+ * User's data using the Google Books API.
+ * 
+ * Additionally, the Book List object also stores collaboratorsIDs.
+ * 
+ * The 'collaboratorsIDs' variable purpose is to allow the implemation of a
+ * sharing feature between Book Lists.
+ * 
+ */
+public final class BookList {
+
+  private final long id = new Random().nextLong();
```

Comment:
Should we make this a constructor variable we can pass in like @steven-solar did in his PR since we don't have a method to store and create these ids yet?

Reply:
Ok